README.md

```diff
@@ -8,27 +12,62 @@ use of an index parameter. The split step is designed to divide the genome into
 chunks of adjustable size to optimise for runtime/memory usage requirements.
 For simple execution of CaVEMan please see [cgpCaVEManWrapper](https://github.com/cancerit/cgpCaVEManWrapper)
 
-| Master                                                                                                              | Dev                                                                                                              |
-| ------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
-| [![Build Status](https://travis-ci.org/cancerit/CaVEMan.svg?branch=master)](https://travis-ci.org/cancerit/CaVEMan) | [![Build Status](https://travis-ci.org/cancerit/CaVEMan.svg?branch=dev)](https://travis-ci.org/cancerit/CaVEMan) |
+---- 
+<!-- TOC depthFrom:2 -->
+
+- [Installation](#installation)
+    - [Default install](#default-install)
+    - [Dependencies](#dependencies)
+- [Prerequisites to running CaVEMan](#prerequisites-to-running-caveman)
+- [Optional inputs (will result in more accurate calls)](#optional-inputs-will-result-in-more-accurate-calls)
+- [Processing flow](#processing-flow)
+    - [Setup](#setup)
+    - [Split](#split)
+    - [Mstep](#mstep)
+    - [Merge](#merge)
+    - [Estep](#estep)
+- [File Formats](#file-formats)
+    - [Copy Number](#copy-number)
+    - [Ignored regions file](#ignored-regions-file)
+- [Default Settings](#default-settings)
+- [LICENCE](#licence)
+
+<!-- /TOC -->
+
+## Installation
 
-Installation
-============
+### Default install
 
-See INSTALL.TXT
+Use the [setup.sh](setup.sh) script. This installs htslib temporarily and uses the autoconf/autotools method to install
+CaVEMan binaries to `~/install_loc/bin/`
 
-## Prerequisites
+`./setup.sh ~/install_loc`
 
-* As of version 1.6.0 CaVEMan supports cram files (with index).
-* BWA Mapped, indexed, duplicate marked/removed bam files, for both a normal and tumour sample
-* Reference.fasta and index
-* A one based bed style format file of regions to ignore during analysis (see specified format).
-* [zlib](https://zlib.net/) >= 1.2.3.5
+### Dependencies
+
+CaVEMan is designed to run in a compute farm/clustre environment.
+
+CaVEMan depends on [htslib], by default [htslib] is installed temporarily and linked for you by [setup.sh](/setup.sh)
+
+CaVEMan also requires the following libraries to be installed
+
+- build-essential
+- autoconf
+- pkg-config
+- check 
+- [zlib](https://zlib.net/) > 1.2.3.4
```

Comment:
During many-heads (@keiranmraine, @kathryn-beal, @AndyMenzies) debugging of canPipe_test install failure, possibly involving the `(zlib > x)` vs `(zlib >= x)` issue, I observed that dependency versions appear in multiple places here and I was considering mentioned it in this review.

I don't think the autoconfing has caused it, but it is an untidiness or potential issue which now seems to be causing problems.

One possible (but quirky) solution would be to grep version deps out of README.md or another smaller doc, using some grubby shellscript that gives a unified interface to dependency names/versions/OS-package-names.  Another wheel to invent, for where pkg-config is dragging along the ground?  `8-(`

Reply:
https://github.com/cancerit/CaVEMan/commit/35bebe55501e2c6262667ac071878f10aa4f3087 Removes the check from setup.sh. Docs in process of being improved to show requirements for various flavours, however once the docker image is complete this will be less relevant.

Reply:
Tested on Centos7, Xenial and Bionic